src/app/routes/app-routes.tsx

```diff
@@ -138,13 +158,14 @@ function AppRoutesAfterUserHasConsented() {
         />
         <Route
           path={RouteUrls.SetPassword}
-          element={
-            <OnboardingGate>
-              <SetPasswordPage />
-            </OnboardingGate>
-          }
+          lazy={async () => {
```

Comment:
Curious why you are doing this now just so I know?

Reply:
To fix FF bundle issues. We crossed the 4MB mark, and so needed to get bundle size down, this seemed like best approach. Can use build:analyze to see difference